src/dtos/unit.dto.ts

```diff
@@ -93,19 +100,110 @@ export class UnitDTO {
+    description: propertyMetadata.unitDTOCommOpDate.description,
+    example: propertyMetadata.unitDTOCommOpDate.example,
+    name: propertyMetadata.unitDTOCommOpDate.fieldLabels.value,
   })
-  nonLoadBasedIndicator: number;
+  @IsOptional()
+  @IsDateString()
+  commOpDate?: string;
+
+  @ApiProperty({
+    description: propertyMetadata.unitDTOComrOpDate.description,
+    example: propertyMetadata.unitDTOComrOpDate.example,
+    name: propertyMetadata.unitDTOComrOpDate.fieldLabels.value,
+  })
+  @IsOptional()
+  @IsDateString()
+  comrOpDate?: string;
+
+  @ApiProperty({
+    description: propertyMetadata.unitDTOUserId.description,
+    example: propertyMetadata.unitDTOUserId.example,
+    name: propertyMetadata.unitDTOUserId.fieldLabels.value,
+  })
+  @IsNotEmpty()
+  @IsString()
+  userId: string;
+
+  @ApiProperty({
+    description: propertyMetadata.unitDTOAddDate.description,
+    example: propertyMetadata.unitDTOAddDate.example,
+    name: propertyMetadata.unitDTOAddDate.fieldLabels.value,
+  })
+  @IsNotEmpty()
+  @IsDateString()
+  addDate: string;
+
+  @ApiProperty({
+    description: propertyMetadata.unitDTOUpdateDate.description,
+    example: propertyMetadata.unitDTOUpdateDate.example,
+    name: propertyMetadata.unitDTOUpdateDate.fieldLabels.value,
+  })
+  @IsOptional()
+  @IsDateString()
+  updateDate?: string;
+
+  @ApiProperty({
+    description: propertyMetadata.unitDTOOpStatusCd.description,
+    example: propertyMetadata.unitDTOOpStatusCd.example,
+    name: propertyMetadata.unitDTOOpStatusCd.fieldLabels.value,
+  })
+  @IsOptional()
+  @IsString()
+  opStatusCd?: string;
+
+  @ApiProperty({
+    description: propertyMetadata.unitDTOStatusBeginDate.description,
+    example: propertyMetadata.unitDTOStatusBeginDate.example,
+    name: propertyMetadata.unitDTOStatusBeginDate.fieldLabels.value,
+  })
+  @IsOptional()
+  @IsDateString()
+  statusBeginDate?: string;
+
+  @ApiProperty({
+    description: propertyMetadata.unitDTOUnitTypeCd.description,
+    example: propertyMetadata.unitDTOUnitTypeCd.example,
+    name: propertyMetadata.unitDTOUnitTypeCd.fieldLabels.value,
+  })
+  @IsOptional()
+  @IsString()
+  unitTypeCd?: string;
+
+  @ApiProperty({
+    description: propertyMetadata.unitDTOAuditUser.description,
+    example: propertyMetadata.unitDTOAuditUser.example,
+    name: propertyMetadata.unitDTOAuditUser.fieldLabels.value,
+  })
+  @IsOptional()
+  @IsString()
+  auditUser?: string;
+
+  @ApiProperty({
+    description: propertyMetadata.unitDTOAuditDate.description,
+    example: propertyMetadata.unitDTOAuditDate.example,
+    name: propertyMetadata.unitDTOAuditDate.fieldLabels.value,
+  })
+  @IsOptional()
+  @IsDateString()
+  auditDate?: string;
+
+  @ApiProperty({
+    description: propertyMetadata.unitDTOActive.description,
+    example: propertyMetadata.unitDTOActive.example,
+    name: propertyMetadata.unitDTOActive.fieldLabels.value,
+  })
+  @IsOptional()
+  @IsBoolean()
+  active?: boolean;
```

Comment:
Not sure what active is.

Reply:
This is selected in the `getUnitDetails` method of `UnitService` and `UnitWorkspaceService`:
```sql
            true as "active"
```
I haven't found where it's used, yet.

Reply:
This is selected in the `getUnitDetails` method of `UnitService` and `UnitWorkspaceService`:
```sql
            true as "active"
```

I'm not yet sure where it is used.